src/components/Verify.js

```diff
@@ -322,10 +340,18 @@ const Verify = (props) => {
     : [];
 
   const captureImageItems = captureImages
+
     .concat(placeholderImages)
     .map((capture) => {
       return (
-        <Grid item xs={12} sm={6} md={4} xl={3} key={capture.id}>
+        <Grid
+          item
+          xs={showBigSize ? 0 : 12}
```

Comment:
I suspect responsive grid sizing is too limiting here. It only adjusts to the viewport width, so doesn't account for the fixed-width tagging panel.
I suggest doing your own calculation for items per row based on the container width and setting the item width to `calc(100% / ${numCaptures}`.

See https://gist.github.com/nmcharlton/0bee3372c2eec090ee5b0d97342e0c11 for a possible solution.